_guides/integrations-3.md

```diff
@@ -1,30 +1,35 @@
 ---
 layout: page
-title: Integrations Part 3 – Sending Data directly to OpenLattice's Servers
-description: Bypass manual transfer of .csv files by connecting your database directly to OpenLattice. This guide is recommended for individuals who are interested in setting up integrations into the OpenLattice platform where data is transferred directly and securely from database to database. Reach out to support@openlattice.com if you need any assistance with your integration.
+title: Integration Pipeline: Secure Data Transfers & Speed
+description: Maximize both security and performance by using our **Launchpad** application to transfer data directly and securely from your database directly into OpenLattice. This is the recommended data pipeline for using the OpenLattice platform. Reach out to support@openlattice.com if you need any assistance with your integration.
```

Comment:
`directly and securely [...] directly into`, using "directly" twice is redundant. Can you rephrase? Perhaps `... to transfer data from your database directly and securely into OpenLattice`?